plugins/azure/modelgauge/suts/azure_client.py

```diff
@@ -140,9 +140,21 @@ def scope(cls) -> str:
         return "azure_phi_3_5_mini_endpoint"
 
 
+class PhiMoEKey(AzureApiKey):
+    @classmethod
+    def scope(cls) -> str:
+        return "azure_phi_3_5_moe_endpoint"
+
+
 SUTS.register(
     AzureChatSUT,
-    "phi-3.5-mini",
+    "phi-3.5-mini-instruct",
     "https://Phi-3-5-mini-instruct-hfkpb.eastus2.models.ai.azure.com",
     InjectSecret(PhiMiniKey),
 )
+SUTS.register(
+    AzureChatSUT,
+    "phi-3.5-moe-instruct",
+    "https://Phi-3-5-MoE-instruct-abneb.eastus2.models.ai.azure.com",
```

Comment:
Do we want to make this a secret? That way it's not hardcoded.

Reply:
I think it's fine to hardcode unless its actually a "secret"

Reply:
We really need to run these tests, so let's hardcode it for now. In the future we should talk about config that isn't secrets.